modules/custom/az_publication/az_publication.install

```diff
@@ -83,3 +86,159 @@ function az_publication_update_1020702() {
   }
 
 }
+
+/**
+ * Adds field_az_publication_mapped_type field.
+ */
+function az_publication_update_1020901() {
+  // The path to your module's config/install directory.
+  $module_path = \Drupal::service('extension.list.module')->getPath('az_publication') . '/config/install';
+
+  // Create a new FileStorage object.
+  $source_storage = new FileStorage($module_path);
+
+  // Import the specific configurations.
+  $field_storage_config = $source_storage->read('field.storage.node.field_az_publication_mapped_type');
+  \Drupal::entityTypeManager()->getStorage('field_storage_config')->create($field_storage_config)->save();
```

Comment:
This update seems to cause an error if the field already exists (for example, if they've already run a distribution update)

```
>  [notice] Update started: az_publication_update_1020901
>  [error]  'field_storage_config' entity with ID 'node.field_az_publication_mapped_type' already exists. 
>  [error]  Update failed: az_publication_update_1020901 
 [error]  Update aborted by: az_publication_update_1020901 
 [error]  Finished performing updates. 
```

Reply:
I think this is resolved now.